packages/office-addin-dev-certs/test/test.ts

```diff
@@ -143,7 +143,7 @@ describe("office-addin-dev-certs", function() {
                 sandbox.stub(verify, "isCaCertificateInstalled").returns(false);
                 await install.installCaCertificate(testCaCertificatePath, machine);
                 assert.strictEqual(execSync.callCount, 1);
-                assert.strictEqual(execSync.calledWith(`powershell -ExecutionPolicy Bypass scripts\\install.ps1 LocalMachine '${testCaCertificatePath}'`), true);
+                assert.strictEqual(execSync.calledWith(`powershell -ExecutionPolicy Bypass ${path.resolve(__dirname, "..\\scripts\\install.ps1")} LocalMachine '${testCaCertificatePath}'`), true);
```

Comment:
You could move the path to the script file to a `const script = ....` line at a higher scope so the variable can be shared for all of the test cases.